src/main/resources/static/assets/js/sdh.js

```diff
@@ -185,20 +168,20 @@ class ModalEditRow {
 				url: contextPath + '/api/clinician_priority/',
 				contentType: 'application/json',
 				success: data => {
-					this.priorityOptions = data._embedded.clinicianPriorities.map(it => {
-						return {value: it._links.self.href, text: it.description};
+					this.priorities = data._embedded.clinicianPriorities.map(it => {
+						return {href: it._links.self.href, description: it.description};
 					});
-					this.priorityOptions.unshift({value:null, text:this.emptyOption});
+					this.priorities.unshift({href:null, description:this.emptyOption});
```

Comment:
The code is fine as it is, but you could do this using the spread operator instead:

```js
this.priorities = [
  {href: null, description: this.emptyOption},
  ...data._embedded.clinicianPriorities.map(it => ({href: it._links.self.href, description: it.description}))
];
```

Reply:
Nice. I think I'd used this operator before but had forgotten about it. I always liked Groovy's way of doing this list << a << b